### DISCUSSION TITLE
Trying to understand " security_level" parameter?

### DISCUSSION THREAD

**Discussion by GPU-server** [Comfy-Org/ComfyUI-Manager#1132] [Q&A]
Hi, I encoutered instances where I got a message about the security level  not allowing me to download nodes.
Would you say it is okay to reduce the security level TEMPORARILY to download nodes from "entrusted somewhat" custom node repos, then raise the level of security back to normal?

**Comment by ltdrdata** [comment]
Installing custom nodes from the default channel at the normal security level should not be a problem.
Be aware that temporarily lowering the security level does temporarily make your security vulnerable.
